qt/aqt/main.py

```diff
@@ -1112,6 +1120,7 @@ def onStudyKey(self) -> None:
     ##########################################################################
 
     def closeEvent(self, event: QCloseEvent) -> None:
+        self.pm.set_last_run_version()
```

Comment:
I figured `last_run_version` is the most useful, when it is set as late as possible. But is this the right place?

Reply:
The only other place I can think of is at the end of _run() in aqt/__init__.py, but I can't think of a big reason to prefer one over the other, so until one arises, this should do :-)